YoCode.sln

```diff
@@ -3,24 +3,24 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27703.2035
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "YoCode", "YoCode\YoCode.csproj", "{CEE256C1-7ED3-4CD4-AF08-7E606B363922}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "YoCode_XUnit", "Tests\YoCode_XUnit.csproj", "{4CD1AD56-DD80-4323-AFD9-C79220CEA8BD}"
 EndProject
```

Comment:
Did you remove and then re-add the test project for some reason?

Reply:
one of the projects was missing when i first turned on my computer this morning

Reply:
Super weird.